lib/process-content.js

```diff
@@ -21,7 +21,6 @@ module.exports = function processContent(
   // SugarSS support:
   if (ext === ".sss") {
     if (!sugarss) {
-      /* c8 ignore next 3 */
```

Comment:
Why this change?

Reply:
Unsure what my motivation was at the time.
I've reverted this to the previous state.